app/dao/services_dao.py

```diff
@@ -197,8 +197,11 @@ def dao_fetch_live_services_data():
     return results
 
 
-def dao_fetch_service_by_id(service_id, only_active=False):
-    query = Service.query.filter_by(id=service_id).options(joinedload("users"))
+def dao_fetch_service_by_id(service_id, only_active=False, eager_users=True):
```

Comment:
Optional: I might suggest `eager_join_users` is clearer than `eager_users`. It makes a bit more explicit that eager is in the sense of an eager join.

I'll let you decide if you think that is an improvement/worth it